### ISSUE TITLE
Hepatolithiasis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10033]
**Preferred term label:**
Hepatolithiasis

**Synonyms**
Multiple liver intraductal calculi in the absence of extrahepatic biliary obstruction

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0574143

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Recurrent pyogenic cholangitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000063
- label: Hepatolithiasis
- parents: [Abnormal intrahepatic bile duct morphology (HP:0011040)](https://hpo.jax.org/app/browse/term/HP:0011040)
- pmid: PMID:26730152
- def: The presence of gallstones within the intrahepatic bile ducts proximal to the right and left hepatic ducts.
- comment: Hepatolithiasis is benign in nature, but the prognosis is poor due to an association with recurrent cholangitis, biliary strictures, liver abscesses, and atrophy or cirrhosis of the affected liver. Hepatolithiasis is also a known risk factor for intrahepatic cholangiocarcinoma.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10033
- synonyms:
